src/web/nextui/src/app/eval/ResultsView.tsx

```diff
@@ -141,6 +158,43 @@ export default function ResultsView({
     }
   };
 
+  const handleComparisonEvalSelected = async (evalId: string) => {
+    setAnchorEl(null);
+    try {
+      const response = await fetch(`${await getApiBaseUrl()}/api/results/${evalId}`, {
+        cache: 'no-store',
+      });
+      const body = await response.json();
+      const comparisonTable = body.data.results.table as EvaluateTable;
+
+      // Combine the comparison table with the current table
+      const combinedTable: EvaluateTable = {
+        head: {
+          prompts: [...table.head.prompts, ...comparisonTable.head.prompts],
+          vars: table.head.vars, // Assuming vars are the same
+        },
+        body: table.body.map((row, index) => ({
+          ...row,
+          outputs: [...row.outputs, ...(comparisonTable.body[index]?.outputs || [])],
+        })),
+      };
+
+      // Update the state with the combined table
+      setTable(combinedTable);
+
+      // Update other relevant state if needed
+      setConfig({
```

Comment:
We should update the `Outputs` section of the table to denote which column corresponds to which eval